src/desktop/apps/auction/components/layout/auction_info/Registration.tsx

```diff
@@ -81,6 +82,12 @@ const Registration: React.FC<RegistrationProps> = props => {
     shouldPromptIdVerification,
   } = bidderQualifications(auction, user, userRegistration)
 
+  const renderZendeskScript = () => {
+    if (typeof window !== "undefined" && window.zEmbed) return
```

Comment:
suggestion:

seems this check is done every where the component is being used (in [OrderApp](https://github.com/artsy/force/blob/5e102304113da2a26dadbb285526acb872fa070e/src/v2/Apps/Order/OrderApp.tsx#L112) and [ArtworkMeta](https://github.com/artsy/force/blob/5e102304113da2a26dadbb285526acb872fa070e/src/v2/Apps/Artwork/Components/ArtworkMeta.tsx#L161)) so it is possible to move the check inside the component. (is that the case @xtina-starr?)

Reply:
That's a good suggestion @sepans!!

Reply:
I tried this and the `window` object is undefined when not called in the React component this check is being rendered in, I think it is bc it isn't in the func's scope when called this way. If there's a workaround, such as having the function accept `window` as a parm that could work, but it seems hacky.